packages/contracts/contracts/voting/admin-address/AdminAddress.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import {Counters} from "@openzeppelin/contracts/utils/Counters.sol";
+
+import {PluginCloneable} from "../../core/plugin/PluginCloneable.sol";
+import {IDAO} from "../../core/IDAO.sol";
+
+/// @title AdminAddress
+/// @author Aragon Association - 2022.
+/// @notice The Admin address governance plugin, to serve as a DAO's primary full controller.
+contract AdminAddress is PluginCloneable {
+    using Counters for Counters.Counter;
+
+    /// @notice The [ERC-165](https://eips.ethereum.org/EIPS/eip-165) interface ID of the contract.
+    bytes4 internal constant ADMIN_ADDRESS_INTERFACE_ID =
+        this.initialize.selector ^ this.executeProposal.selector;
+
+    bytes32 public constant ADMIN_EXECUTE_PERMISSION_ID = keccak256("ADMIN_EXECUTE_PERMISSION");
```

Comment:
This is the best example of dogfooding, but... wouldn't it be cheaper to just keep note of the admin's address :)

This can be marked as resolved, I just had to say it